[tvOS, visionOS, watchOS] Skip install step for WebKit process extensions

pvollan opened the pull request:
#### 2a81bcc7ebee1e798932ea49c0aaab2ced22015e
<pre>
[tvOS, visionOS, watchOS] Skip install step for WebKit process extensions
<a href="https://bugs.webkit.org/show_bug.cgi?id=276210">https://bugs.webkit.org/show_bug.cgi?id=276210</a>
<a href="https://rdar.apple.com/130775557">rdar://130775557</a>

Reviewed by NOBODY (OOPS!).

Skip install step for WebKit process extensions on platforms where they are not enabled.
We need to explicitly skip the install step for visionOS and watchOS, since they
otherwise will match an iPhone SDK predicate.

* Source/WebKit/Configurations/BaseExtension.xcconfig:
</pre>

aproskuryakov commented on Source/WebKit/Configurations/BaseExtension.xcconfig:
```diff
@@ -29,6 +29,11 @@ CLANG_INSTRUMENT_FOR_OPTIMIZATION_PROFILING = NO;
 INSTALL_PATH = $(INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 INSTALL_PATH[sdk=*simulator] = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
 INSTALL_PATH[sdk=iphone*17.4*] = $(WK_INSTALL_PATH_PREFIX)/System/Library/ExtensionKit/Extensions;
+SKIP_INSTALL = NO;
```
Would it be more appropriate to do the opposite, skipping by default, and unskipping on platforms where it's enabled?

pvollan replied:
Ah, good point, will fix!

Thanks for reviewing, all!

pvollan committed:
[tvOS, visionOS, watchOS] Skip install step for WebKit process extensions
https://bugs.webkit.org/show_bug.cgi?id=276210
rdar://130775557

Reviewed by Alexey Proskuryakov and Elliott Williams.

Skip install step for WebKit process extensions on platforms where they are not enabled.

* Source/WebKit/Configurations/BaseExtension.xcconfig:

Canonical link: https://commits.webkit.org/280658@main
